nncf/experimental/torch/fx/nncf_graph_builder.py

```diff
@@ -120,15 +135,15 @@ def create_nncf_graph(model: torch.fx.GraphModule) -> PTNNCFGraph:
 
         for source_node in model.graph.nodes:
             node_type, node_metatype = GraphConverter._get_node_type_and_metatype(source_node, model)
```

Comment:
Could you suggest how to take function namespace into account when defining metatype?
cc' @daniil-lyakhov @AlexanderDokuchaev

Reply:
Alright, I will check for possible ways to map with function namespaces as well.